test/org/sagebionetworks/bridge/services/SendMailViaAmazonServiceTest.java

```diff
@@ -0,0 +1,80 @@
+package org.sagebionetworks.bridge.services;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.when;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import org.sagebionetworks.bridge.exceptions.BridgeServiceException;
+import org.sagebionetworks.bridge.models.studies.EmailTemplate;
+import org.sagebionetworks.bridge.models.studies.MimeType;
+import org.sagebionetworks.bridge.models.studies.Study;
+import org.sagebionetworks.bridge.services.email.BasicEmailProvider;
+
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceClient;
+import com.amazonaws.services.simpleemail.model.SendRawEmailResult;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SendMailViaAmazonServiceTest {
+
+    private static final String SUPPORT_EMAIL = "email@email.com";
+    private static final String RECIPIENT_EMAIL = "recipient@recipient.com";
+    
+    private SendMailViaAmazonService service;
+    
+    @Mock
+    private AmazonSimpleEmailServiceClient emailClient;
+    
+    @Mock
+    private EmailVerificationService emailVerificationService;
+    
+    @Mock
+    private SendRawEmailResult result;
+    
+    @Before
+    public void before() {
+        service = new SendMailViaAmazonService();
+        service.setEmailClient(emailClient);
+        service.setEmailVerificationService(emailVerificationService);
+    }
+    
+    @Test
+    public void unverifiedEmailThrowsException() {
+        when(emailVerificationService.isVerified(SUPPORT_EMAIL)).thenReturn(false);
+        
+        BasicEmailProvider provider = new BasicEmailProvider.Builder()
+                .withStudy(Study.create())
+                .withRecipientEmail(SUPPORT_EMAIL)
```

Comment:
This test doesn't do what you think it does. You created a blank study, then tried to send the email to SUPPORT_EMAIL. EmailVerificationService tries to verify that that a blank email address is verified. This throws, but for the wrong reason.